dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -435,14 +441,21 @@ struct CollectionsShowArgs {
 }
 
 #[derive(Debug, Args, Clone, Copy)]
-struct PhysicalDisksArgs {
+struct InvPhysicalDisksArgs {
```

Comment:
A disk in the inventory is a different thing than a "control plane physical disk", so this was renamed to be more explicit.